src/Agent/NewRelic/Agent/Core/Attributes/AttributeDefinitionService.cs

```diff
@@ -281,6 +283,19 @@ public AttributeDefinition<object, object> GetFaasAttribute(string name)
         }
 
 
+        private AttributeDefinition<object, object> CreateCloudSdkAttribute(string attribName)
```

Comment:
This is only slightly different than `CreateFaasAttribute()` (which applies the attribute to the transaction as well as trace and span)-- do we really need the distinction?

Reply:
I think so: `Faas` attributes are `Intrinsic`s, while the Cloud attributes are `Agent` attributes, and aren't supposed to apply to TransactionEvents.